driver/normalizer/uast.go

```diff
@@ -0,0 +1,66 @@
+package normalizer
+
+import (
+	"github.com/bblfsh/java-driver/driver/normalizer/jdt"
+
+	"github.com/bblfsh/sdk/uast"
```

Comment:
I have not been able to find package "github.com/bblfsh/sdk/uast" so I am quite lost here and cannot really review this PR.  Are you using a different version of this package that the one I can find in github?